src/main/java/nl/tudelft/ewi/devhub/server/database/AssignTAs.java

```diff
@@ -0,0 +1,54 @@
+package nl.tudelft.ewi.devhub.server.database;
```

Comment:
Could we do something where each TA has a field indicating how many groups they can be assigned to? I can imagine that not each TA has an equal amount of time for grading for example.